pygmt/datasets/tile_map.py

```diff
@@ -79,6 +88,14 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
         will stop trying to fetch more tiles from a rate-limited API [Default
         is ``2``].
 
+    zoom_adjust : int or None
+        Optional. The amount to adjust a chosen zoom level if it is chosen
+        automatically. Values outside of -1 to 1 are not recommended as they
+        can lead to slow execution. [Default is ``None``].
```

Comment:
Rewrap the newly added docstrings to 88 characters?

Reply:
Done at c36deff7b384c8f82ef26d6cab6a3d4ec9083845